apps/documize/openshift/backup/openshift/backup-deploy.param

```diff
@@ -0,0 +1,38 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: backup
+# Template File: templates/backup-deploy.json
+#=========================================================
+NAME=backup
+SOURCE_IMAGE_NAME=backup
+IMAGE_NAMESPACE=hmg6pw-prod
+TAG_NAME=
+# secrete name:
+DATABASE_DEPLOYMENT_NAME=
+# should be corresponding to the db host name:
+SSO_PGSQL_MASTER_USER_KEY_NAME=
+SSO_PGSQL_MASTER_PASSWORD_KEY_NAME=
```

Comment:
not clear on what these are?

Reply:
they are just the database user credential. I renamed it to match the dc for keycloak. You could rename it to general parameters or match the dc for documize.